clogMonitor/src/screens/LoginScreen.css

```diff
@@ -49,7 +14,9 @@
   font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
   background-color: #A9A9A9;
 }
-.outlined-input {
+
+input[type="text"],
```

Comment:
Use a class instead of altering the input type because when you alter the style on the input with type text (or any type) you change it for the entire app, which affects other people's styling

Reply:
This was from an earlier version of the code, updated to the newest version of main and fixed.